draft-bbs-signatures.md

```diff
@@ -382,27 +380,25 @@ This operation checks if a public key is valid.
 As an optimization, implementations MAY cache the result of KeyValidate in order to avoid unnecessarily repeating validation for known public keys.
 
 ```
-result = KeyValidate(PK)
+result = KeyValidate(W)
 
 Inputs:
 
-- PK (REQUIRED), an octet string of the form outputted by the SkToPk operation.
+- W (REQUIRED), a public key as calculated at step one (1) in SkToPk.
```

Comment:
I'd prefer we only hand the valid output of SkToPk as the input here

Reply:
I would prefer that as well. That said it's a trade-off. The problem is that the output of `SkToPk` is an octet string, meaning we will need to de-serialize it in `KeyValidate`. Any time we call `KeyValidate` in a procedure though, we de-serialize the PK again. So we are doing two de-serializations wile we only need one.

Maybe define `W` as a point in `G2` will be fine here??

Reply:
Apologies I meant to say `octets_to_point_g2` instead of `SkToPk` so we would handle that deserialization in the procedure that is going to call `KeyValidate`